interop-tests/Dockerfile.native

```diff
@@ -1,6 +1,5 @@
 # syntax=docker/dockerfile:1.5-labs
-FROM rust:1.67.0 as chef
```

Comment:
We need to make sure we have at least Rust 1.67 in the base image.

Reply:
That's the error:

```
#19 251.5 error: package `libp2p-identity v0.2.8` cannot be built because it requires rustc 1.73.0 or newer, while the currently active rustc version is 1.67.0
#19 251.5 Either upgrade to rustc 1.73.0 or newer, or use
#19 251.5 cargo update -p libp2p-identity@0.2.8 --precise ver
#19 251.5 where `ver` is the latest version of `libp2p-identity` supporting rustc 1.67.0
```